### ISSUE TITLE
Research: Most popular messaging apps

### ISSUE THREAD

**Issue by autonome** [autonome/web-and-messaging#2]
*Research:* What are the most popular messaging apps?

* By country
* By userbase size

**Comment by autonome** [comment]
First run through some online research, mostly focused on Asia.

Top
* WeChat (cn, hk, in, ml)
* Line (jp, th, tw)
* Messenger (hk, )
* Facebook
* Viber
* WhatsApp (ml, sg, hk, in)
* Kakao Talk
* Tango
* Kik
* GroupMe
* QQ
* Sina Weibo
* Twitter
* Nimbuzz
* Between
* Zalo
* Path
* Chikka (ph)
* Comm (jp)
* Cubie (tw)
* Pinger
* Hike
* BBM
* MessageMe
* Maaii
* Rounds
* ChatON
* Voxer
* iMessage
* Skype
* Weibo
* Telegram
* Hangouts
* Slack
* Wire
* Signal
* Riot/Matrix
* Snapchat

References for links I got these from in issue #7.

**Comment by Vinosri844** [comment]
Most popular messenger apps in India
        

       Hike
       Facebook
       WhatsApp
       Twitter
       WeChat
       Line
       MessageMe
       Viber
       Nimbuzz

**Comment by autonome** [comment]
Thanks @snsvino! Is this your experience, or did you look this up
somewhere? Would be cool to have a reference. Also, is that in order of
popularity?

**Comment by Vinosri844** [comment]
yeah sir!! this is my experience, but,it is not in popularity order..

**Comment by TharinduDilshan** [comment]
There are many popular messaging apps in Sri Lanka.
Whatsapp
Viber
Facebook Messenger 
Telegram Messenger
Instagram
Twitter 

I have used those apps in my smart phone.

**Comment by Kevin-Kinyua** [comment]
BRIDGIFY;  https://play.google.com/store/apps/details?id=me.bridgefy.main&hl=en
SIGNAL OFFLINE MESSENGER: https://play.google.com/store/apps/details?id=com.raxis.signalapp&hl=en

The above web messenger apps are good to apply on your web pages and more so they work offline.

**Comment by vschs007** [comment]
I think whatsapp is having the maximum user database in asia
